x-pack/plugins/cases/public/containers/use_bulk_create_attachments.tsx

```diff
@@ -43,15 +43,15 @@ const dataFetchReducer = (state: NewCommentState, action: Action): NewCommentSta
 
 export interface PostComment {
   caseId: string;
-  data: CommentRequest;
+  data: CommentRequest | BulkCreateCommentRequest;
```

Comment:
@academo Do you think it will be better to only accept `BulkCreateCommentRequest` (an array of attachments)? It is an easy change.

Reply:
If it is an easy change, I say let's go for it.